media/webviewslim.css

```diff
@@ -2,6 +2,12 @@
   cursor:crosshair;
 }
 
+.poweredByDidact {
+  background: url(../icon/poweredByDidact-larger1-stroke-and-fill.svg);
```

Comment:
the file mentioned here has not been pushed in teh Pull request

Reply:
This CSS change will be removed, so not a biggie. I'll get rid of it.